content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -19,9 +19,16 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
 
 The following methods exist for installing kubectl on Linux:
 
-- [Install kubectl binary with curl on Linux](#install-kubectl-binary-with-curl-on-linux)
-- [Install using native package management](#install-using-native-package-management)
-- [Install using other package management](#install-using-other-package-management)
+- [{{% heading "prerequisites" %}}](#-heading-prerequisites-)
+- [Install kubectl on Linux](#install-kubectl-on-linux)
+  - [Install kubectl binary with curl on Linux](#install-kubectl-binary-with-curl-on-linux)
+  - [Install using native package management](#install-using-native-package-management)
```

Comment:
I don't know where we're at with replacing the word "native" with better words like "built-in" (@sftim / @divya-mohan0209 ?)

Reply:
That sounds like a good idea to me, Shannon. Unfortunately, we will have to ease it since the existing phrasing is used more frequently than we'd like it to be. Built-in package managers haven't gotten a lot of hits when it comes to OSes on a google search as opposed to native package management.

Reply:
We use ”built in” for features included in Kubernetes. I don't think _native_ is a problem word; if it is, the CNCF (sic) needs to lead on fixing that.

Reply:
This is not a PR blocker, just an observation:

I agree with you Tim about clarity and also think regardless of whether we say built in or native we should be clear about which part of the system the feature is native/built in to.

So instead of "native package manager", something like "package manager native to the operating system" or "OS-native package manager" or even just "operating system package manager".

Obviously that's more words but imo makes it clearer to those who might not know right away

Reply:
Understood, will resolve this then